internal/service/drs/generate.go

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+//go:generate go run ../../generate/servicepackage/main.go
+// ONLY generate directives and package declaration! Do not add anything else to this file.
```

Comment:
Consider adding

```go
//go:generate go run ../../generate/tagstests/main.go
```

so that resources have generated tagging tests from the start